### DISCUSSION TITLE
Brink (WIP)

### DISCUSSION THREAD

**Discussion by Jigoku** [redeclipse/base#1207] [Maps and Content Showcase]
### Brink

Still early work in progress as geometry is still being worked on/retextured in many places, most of the inside isn't done yet but i'll post the current state of this now; you can download it below to see if there's any problems with the layout, any feedback / criticism is welcome.

* **setting**
 a refuelling platform, on the edge of a cliffside next to the ocean
* **size**
 medium/large (depending on mode)
* **mode**
 ideal for 8-15 ffa / survivor
 (will hopefully add map-balanced team mode support eventually)
* **variants**
 mid day / moonlight

I won't add a todo / unfinished list of things yet, as it still needs more work. Note, bots will probably jump into the ocean often, as the aiclip and waypoints will need to be redone.
 
## Download 
**r725** - [brink.zip](https://github.com/redeclipse/world/files/5043587/brink.zip)
**r986** - [brink-r986.zip](https://github.com/redeclipse/world/files/5213341/brink-r986.zip)
**r1315** -  https://ufile.io/18dirg19

(just place the zip folder in your local data folder, no need to extract)

![2020-08-07-210347_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/89687943-06c12900-d8f9-11ea-912f-b4e89f20097e.png)
![2020-08-07-210421_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/89687948-0759bf80-d8f9-11ea-8236-2de482eba91c.png)

**Comment by Calinou** [comment]
Looks great!

Unrelated to the map itself, but I can notice on the screenshots that the water pattern is a bit too repetitive. Can we add a variable to change its scale or replace the texture with an higher-resolution variant that spans a larger area?

**Comment by Jigoku** [comment]
**Updated to r986**
* too many changes to list

still needs some retexturing / geometry fixes / lighting fixes / materials

**Comment by Ulukai55** [comment]
Looking excellent! Very nice texture work, I love the orange accents you created. 
And those rock parties are shaped with a lot of care I can tell. Well done!

**Comment by Jigoku** [comment]
This map is starting to dwindle because it's taking so long to finish lol, starting to wonder if the idea was a bit over ambitious, but i'll still finish it up (or change parts of it) depending on any feedback. Lacking a bit of motivation lately.

Notably the lighting needs to be tweaked to compensate for hdr as it can look overbright/dull in some outside areas, the interior lighting is mostly just placeholder/quickly thrown around for now. Still some fair amount of geometry/texturing to be fixed also. Ignore most of the entities for now.

**r1315** -  https://ufile.io/18dirg19

![2021-02-11-022930_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/107597612-12c4d500-6c13-11eb-8dbb-110b58522b80.png)
![2021-02-11-022954_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/107597618-148e9880-6c13-11eb-8a3c-89dd5230ee99.png)
![2021-02-11-023021_1920x1080_scrot](https://user-images.githubusercontent.com/1535179/107597619-15bfc580-6c13-11eb-84b7-d6131bc4cbd7.png)

**Comment by MirceaKitsune** [comment]
r1315 download link is broken, gonna check out r986 in the meantime. This looks like such an amazing map, feels a lot like the Overwatch tutorial level and its style! Has any progress been made and do we have any hope of seeing it finalized this year?

**Comment by MirceaKitsune** [comment]
Just tried `r986` in latest GIT master of RE. It causes an engine crash to desktop with a strange error message in unreadable font.

![Screenshot_20220911_235548](https://user-images.githubusercontent.com/825418/189548531-54ddca09-073a-4479-81d9-c829c2f1db3a.png)

@Jigoku Could you upload the latest version to a different host that doesn't ask for premium subscriptions please? Or if you feel the map is ready now, make a PR for it? This could be an amazing map for RE 2.1 if it makes it on time.

**Reply by Jigoku** [reply]
Yeah, ignore the premium thing, in retrospect it's always a bad idea using those free file upload sites as they expire after 30 days or so. 

That error is weird... but i think a few people have had that before including myself (albeit not related to the map itself). I can't remember what potentially causes it. 

As for the map, it's in a bit of a butchered state atm, i will hopefully try clean it up and repost it soon enough. There were some issues with it now as for the map previously being designed around a larger playerscale at the time, so some areas need to be made a tad smaller so it doesn't feel "empty".

I will try work on it again soon :+1:

**Comment by Jigoku** [comment]
Slow to update here, but forgot about it completely. These were the latest files btw: https://drive.google.com/file/d/1pKflBsZVylSPxDD4wOrvunOwzkeEcjd3/view?usp=sharing
